src/communication/azext_communication/aaz/latest/communication/email/_list.py

```diff
@@ -16,6 +16,10 @@
 )
 class List(AAZCommand):
     """List requests to list all resources in a subscription.
+
+    :example: Get all resources from a subscription/resource group
+        az communication email list --subscription SubscriptionId
+        az communication email list -g ResourceGroup
```

Comment:
actually the --subscription is optional. The command will use login subscription as default. BTW, we usually separate it into two examples like this:

```suggestion
    :example: Get all resources from a subscription
        az communication email list --subscription SubscriptionId

     :example: Get all resources from a resource group
        az communication email list -g ResourceGroup
```

Reply:
separated examples as suggested.